EgammaAnalysis/ElectronTools/interface/EnergyScaleCorrection_class.hh

```diff
@@ -0,0 +1,208 @@
+#ifndef EnergyScaleCorrection_class_hh
```

Comment:
Hi David - Please forget about my previous comment. I had another look and saw that this file is new, so there should be no problem. I have just renamed the file.